### PR TITLE
magento/magento2#27500 - Replace deprecated constructs from OfflinePa…

### PR DESCRIPTION
**Pull request by cristiano-pacheco**
…yments PHPUnit.

Magento is slowly moving towards the latest dependencies in the PHP world. Part of this is to have (at least) supported version of PHPUnit ( https://phpunit.de/supported-versions.html )

### Description (*)
Updated the tests of Offline_Payments to be compatible with PHPUnit 8.

1. magento/magento2#27500 - app/code/OfflinePayments item.

### Manual testing scenarios (*)

```
 vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist app/code/Magento/OfflinePayments
```

### CODE REVIEW

File: app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php [hunk 1]
```diff
@@ -3,22 +3,24 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
```
Comment (lbajsarowicz) [inline app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php:7]: You can introduce `declare(strict_types=1);`

File: app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php [hunk 2]
```diff
@@ -44,18 +46,18 @@ protected function setUp()
     }
 
     /**
-     * @covers \Magento\OfflinePayments\Block\Info\Checkmo::getPayableTo
+     * @covers       \Magento\OfflinePayments\Block\Info\Checkmo::getPayableTo
```
Comment (lbajsarowicz) [inline app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php]: You don't need indent there.

File: app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php [hunk 3]
```diff
@@ -73,18 +75,18 @@ public function getPayableToDataProvider()
     }
 
     /**
-     * @covers \Magento\OfflinePayments\Block\Info\Checkmo::getMailingAddress
+     * @covers       \Magento\OfflinePayments\Block\Info\Checkmo::getMailingAddress
```
Comment (lbajsarowicz) [inline app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php]: Unnecessary indent

File: app/code/Magento/OfflinePayments/Test/Unit/Model/CheckmoConfigProviderTest.php [hunk 4]
```diff
@@ -3,34 +3,45 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
```
Comment (lbajsarowicz) [inline app/code/Magento/OfflinePayments/Test/Unit/Model/CheckmoConfigProviderTest.php:7]: Introduce strict types.

**Review by lbajsarowicz — APPROVED** [review]
Just minor changes. You can leave as it is, but if you have extra 5 minutes, just review what I suggest.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7271](https://jira.corp.magento.com/browse/ENGCOM-7271) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by magento-engcom-team** [comment]
@cristiano-pacheco thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php (hunk 1 above)
Comment (cristiano-pacheco) [reply app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php:7]: done.

File: app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php (hunk 2 above)
Comment (cristiano-pacheco) [reply app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php]: done.

File: app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php (hunk 3 above)
Comment (cristiano-pacheco) [reply app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php]: done.

File: app/code/Magento/OfflinePayments/Test/Unit/Model/CheckmoConfigProviderTest.php (hunk 4 above)
Comment (cristiano-pacheco) [reply app/code/Magento/OfflinePayments/Test/Unit/Model/CheckmoConfigProviderTest.php:7]: done.

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

 vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist app/code/Magento/OfflinePayments

**Before:**
Screenshot from 2020-04-08 09-55-54

**After:**
Screenshot from 2020-04-08 09-53-42

**Comment by engcom-Echo** [comment]
Failed functional tests not related to the changes in this PR

**Pull request merged**